src/server.cpp

```diff
@@ -2575,6 +2592,24 @@ void Server::DenySudoAccess(u16 peer_id)
 	Send(&pkt);
 }
 
+
+void Server::DenyAccessCompliant(u16 peer_id, u16 proto_ver, AccessDeniedCode reason,
```

Comment:
Should this be named something like DenyAccessWithReason?  I think you meant Complaint here, not Compliant, either way.